changelog.txt

```diff
@@ -16,10 +16,16 @@ Template for new versions:
 
 ## New Features
 
-- `gui/design`: add option to draw N-point stars, hollow or filled or inverted, and change the main axis to orient in any direction
+- `gui/design`: add new option to draw N-point stars.
+  - The default has 5 points, use 'B'/'b' to increase/decrease points.
+  - They can be hollow or filled, and inverted.
+  - The next-point offset can be increased/decreased using 'N'/'n' which particularly affects 7 point stars and above to make them spikier or smoother, but can also be used to decrease to 1 to make symmetrical polygons or increase to N which only paints the vertexes.
+  - The orientation can be changed by adding a main axis point using 'v' and moving this to point in the desired direction.
 
 ## Fixes
 
+- `gui/design`: reduced diameter of star as thickness increases to remain within bounding box
```

Comment:
no need for a "fixes" line for the star shape since it hasn't been out in a release yet. However, if this fixes the bulge for the line shape, I'd say that deserves a "fixes" entry.

Reply:
What's the bulge in the line shape issue?

Reply:
Made the horizontal and vertical thickness consistent for curved lines as well, maybe this is the bulge you were referring to.

Reply:
The bulge in the line shape was similar to what was happening for the star. When increasing the line thickness, the designated tiles can bulge beyond the selected endpoint, leading to potentially "selecting" tiles that are outside of the map:
![image](https://github.com/user-attachments/assets/67d734d7-8355-4c39-acaf-99f907e23b90)

This PR fixes the star shape, but the line still suffers from bulging.

I'll merge this PR as-is because the line bulge was a pre-existing issue, but if you have ideas on how to fix that, I'd love to hear them